gpflow/models/util.py

```diff
@@ -12,20 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Callable, Sequence, Union
+from typing import Any, Callable, Sequence, TypeVar, Union
 
 import numpy as np
 import tensorflow as tf
 
+from ..base import AnyNdArray
 from ..config import default_float
 from ..inducing_variables import InducingPoints, InducingVariables
 from .model import BayesianModel
 from .training_mixins import Data, ExternalDataTrainingLossMixin
 
+_I = TypeVar("_I", bound=InducingVariables)
+_InducingILike = Union[_I, tf.Tensor, AnyNdArray]
 
-def inducingpoint_wrapper(
-    inducing_variable: Union[InducingVariables, tf.Tensor, np.ndarray]
-) -> InducingVariables:
+
+InducingVariablesLike = _InducingILike[InducingVariables]
```

Comment:
Do you use `_InducingILike` anywhere else? If not, isn't it less confusing to just define:

```python
InducingVariablesLike = Union[InducingVariables, tf.Tensor, AnyNdArray]
InducingPointsLike = Union[InducingPoints, tf.Tensor, AnyNdArray]
```

Reply:
> Do you use `_InducingILike` anywhere else? If not, isn't it less confusing to just define:
> 
> ```python
> InducingVariablesLike = Union[InducingVariables, tf.Tensor, AnyNdArray]
> InducingPointsLike = Union[InducingPoints, tf.Tensor, AnyNdArray]
> ```

Sure. I used to use it, but then stuff didn't work, and I guess I don't need it anymore.